01_northeast.Rmd

```diff
@@ -0,0 +1,83 @@
+---
+title: Northeast Air Zone
+output:
+  pdf_document:
+    latex_engine: lualatex
+    fig_caption: true
+    includes:
+      in_header: components/header.tex
+mainfont: Arial
+params:
+  ozone: !r ozone_caaqs_results
+  annual_ozone: !r ozone_caaqs$ann_4th_highest
+  pm25: !r pm_caaqs_combined_results
+  annual_pm25: !r pm25_caaqs_annual$yearly_avg
+  stations: !r stations_az
+  airzone: "Northeast"
+---
+
+<!-- Load functions, filter data to airzone -->
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(echo = FALSE, warning = FALSE)
+
+## Load packages, functions, etc.
+source("components/functions.R")
+
+## Filter data to air zone
+airzone <- params$airzone
+ozone <- filter(params$ozone, airzone == !!airzone)
+pm25 <- filter(params$pm25, airzone == !!airzone)
+
+## Filter and add station names to annual ozone
+annual_ozone <- params$annual_ozone %>%
```

Comment:
Because the proper station name is now in the yearly data, you don't need to join with stations to get the names... I think you can filter to the air zone and the stations that are in the final caaqs summary data. This is applicable in all the air zones:

```r
## Filter and add station names to annual ozone
annual_ozone <- params$annual_ozone %>%
  filter(airzone == !!airzone, ems_id %in% ozone$ems_id)

## Filter and add station names to annual pm25
annual_pm25 <- params$annual_pm25%>%
  filter(airzone == !!airzone, ems_id %in% pm25$ems_id)
```

Reply:
The yearly data has the station names but not the air zone names, but selecting based on the ems_ids in the summary data is probably sufficient since that will have already been filtered down to the air zone. So this will become:

```r
annual_ozone <- params$annual_ozone %>%
  filter(ems_id %in% ozone$ems_id)
```

Reply:
Oh you're right - that makes sense, `ems_id` should do the trick.